<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frontend_class</class>
 <widget class="QMainWindow" name="Frontend_class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frontend_class</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Search Algorithmus</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>databaseeditor</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QTableWidget" name="tableWidget">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Hardware</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>oscillator</string>
          </property>
          <widget class="QComboBox" name="comboBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>381</width>
             <height>27</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>30</y>
             <width>161</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>Used oscillator</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>100</y>
             <width>79</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>frequency</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>79</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>stability</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>200</y>
             <width>79</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>aging</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>120</y>
             <width>79</width>
             <height>28</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>170</y>
             <width>79</width>
             <height>28</height>
            </rect>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>220</y>
             <width>79</width>
             <height>28</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>90</y>
             <width>151</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>confirm oscillator</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Network</string>
          </property>
          <widget class="QComboBox" name="comboBox_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>381</width>
             <height>27</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>30</y>
             <width>161</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>Used Network</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Systemparameter</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_min">
          <property name="title">
           <string>MIN</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="spinBoxdouble_min">
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_max">
          <property name="title">
           <string>MAX</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QDoubleSpinBox" name="spinBoxdouble_max">
             <property name="maximum">
              <double>10000.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_7">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Variables support: min/max OR 1 infix string with other variables at Min
!use with caution!dont build endless loops!</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="2" column="2">
            <widget class="QPushButton" name="Autofill">
             <property name="text">
              <string>Autofill</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="deleteparameter">
             <property name="text">
              <string>Remove Parameter</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="addParameter">
             <property name="text">
              <string>Add Parameter</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QTableWidget" name="System_param_table">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Other Variables</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Min</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Max</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Equation</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>Taxonomie</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="5" column="0" colspan="2" alignment="Qt::AlignTop">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Chose Values (x,y,...) EMPTY for everything</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="reload_db">
          <property name="text">
           <string>reload Database</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" alignment="Qt::AlignRight">
         <widget class="QPushButton" name="Taxonomie_load">
          <property name="text">
           <string>load other Taxonomie</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2" alignment="Qt::AlignTop">
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>130</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>130</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="4" column="0">
            <widget class="QRadioButton" name="Opt_Network">
             <property name="text">
              <string>Network utilization</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QRadioButton" name="Opt_wighted_combination">
             <property name="text">
              <string>Find optima of f()=</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="Opt_Precision">
             <property name="text">
              <string>Precision</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="Opt_wight_funktion">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Example:&lt;/p&gt;&lt;p&gt;Pre*2+U&lt;br/&gt;=&amp;gt;minimize (Precision in ms)*2+(Network utilisation)&lt;/p&gt;&lt;p&gt;=&amp;gt;possible Parameter are in Systemparameter tab&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Optimalize</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="Opt_Error">
             <property name="text">
              <string>no optimization</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Listt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Import list</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
